SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorModule.java

```diff
@@ -38,6 +39,7 @@
 
   @Override
   protected void configure() {
+    install(new SingularityServiceModule());
```

Comment:
side note to bringing in this dep, installing this module would also likely start up a number of pollers/db connections/zk connections/etc which we don't want